src/Arcus.Templates.Tests.Integration/WebApi/Swagger/v1/SwaggerDocCorrelationHeadersTests.cs

```diff
@@ -81,7 +81,7 @@ public async Task GetSwaggerDocs_WithoutOpenApiAndCorrelation_ReturnsNoSwaggerDo
         private static OpenApiDocument LoadOpenApiDocument(string json)
         {
             var reader = new OpenApiStringReader();
-            OpenApiDocument document = reader.Read(json, out OpenApiDiagnostic diagnostic);
+            OpenApiDocument document = reader.Read(json, out OpenApiDiagnostic _);
```

Comment:
```suggestion
            OpenApiDocument document = reader.Read(json, out OpenApiDiagnostic diagnostic);
```

I think we recommend not using abbreviations (I know, in this case it's discussable).

Reply:
I think it is confusing to give an out variable a name, if you don't use it later.   It might look like you forgot to do something with it.   When you do not use it, it is explicit that this is the intention.

Reply:
Yes, you have a point. Let's leave it like this.